Auth0/CredentialsManager.swift

```diff
@@ -76,15 +76,16 @@ public struct CredentialsManager {
     }
     #endif
 
+    #if WEB_AUTH_PLATFORM
```

Comment:
**Explanation for unrelated change:**
By moving this above comment section, comment is now visible if we investigate function's signature 🙏